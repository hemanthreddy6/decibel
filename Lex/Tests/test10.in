const freq: float <- 440hz;
HIGHPASS(200hz);
LOWPASS(1000hz);
EQ(freq, -10, 5);