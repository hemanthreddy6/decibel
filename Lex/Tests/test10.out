CONST
IDENTIFIER
:
FLOAT
<-
FLOAT_LITERAL
;
HIGHPASS
(
FLOAT_LITERAL
)
;
LOWPASS
(
FLOAT_LITERAL
)
;
EQ
(
IDENTIFIER
,
-
INT_LITERAL
,
INT_LITERAL
)
;
